### DISCUSSION TITLE
How can I add a column after a certain column in migration?

### DISCUSSION THREAD

**Discussion by positivedeveloper36** [adonisjs/core#3867] [Help]
I try like this ::

```
 public async up() {
    this.schema.alterTable(this.tableName, (table) => {
      table.string('description', 250).after('name').notNullable()
    })
  }
```

So I use `after`. But when I run the script, it doesn't work. The description column is at the end

How can I solve this problem?

Please help. Thanks

**Comment by thetutlage** [comment]
Depends upon the database you are using. As far as I know, only MySQL supports this

**Reply by positivedeveloper36** [reply]
@thetutlage I use pg admin 4. Is there no other way?

**Reply by thetutlage** [reply]
PG admin is not a database. The database is PostgreSQL and if the database itself does not support columns ordering, then there is no way Lucid can do it

**Reply by positivedeveloper36** [reply]
@thetutlage  Okay. So we can't do anything. If we add a new field, it will be located in the last column
